src/views/About.tsx

```diff
@@ -0,0 +1,12 @@
+import * as Styled from './styled';
```

Comment:
we are not using views terminology at all in React.
Looks like About should be a module
src/modules/About/index.tsx

Reply:
done